consensus/consortium/v2/consortium.go

```diff
@@ -203,6 +209,90 @@ func (c *Consortium) Author(header *types.Header) (common.Address, error) {
 	return header.Coinbase, nil
 }
 
+// VerifyBlobHeader verifies a block's header blob and corresponding sidecar, returning BlobSideCars
+func (c *Consortium) VerifyBlobHeader(block *types.Block, sidecars []*types.BlobTxSidecar) (error, *types.BlobSidecars) {
```

Comment:
I find that we do not use output of *types.BlobSidecars whenever we use this function, so I think we can remove types.BlobSidecars{} reconstruction part here.
On the other hand, tx.Hash can be added when we do `writeBlockSidecars` before dump into db.